README.md

```diff
@@ -42,6 +49,12 @@ https://github.com/altontonn/Capston-Project.git
 - Twitter: [Newton Alumasa](https://twitter.com/AlumasaNewton)
 - LinkedIn: [Newton Alumasa](https://linkedin.com/in/NewtonAlumasa)
 
+## Acknowledgements
```

Comment:
- [ ] [optional] @altontonn great work implementing the changes, it would be better to give credit using a sentence. Something similar to the example below.

`hint`
I would like to give credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) for the great design.